copilot/fastapi/app/config.py

```
@@ -0,0 +1,91 @@
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._check_security_settings()
+    
+    def _check_security_settings(self):
+        """Check and warn about insecure settings."""
+        if not self.argo_cd_password:
+            warnings.warn("Argo CD password not set. Please set ARGO_CD_PASSWORD environment variable.")
+        if not self.verify_ssl:
+            warnings.warn("SSL verification is disabled. This is insecure and should not be used in production.")
+
+@lru_cache()
```

Ruff (0.8.2)

Unnecessary parentheses to `functools.lru_cache`

Remove unnecessary parentheses

(UP011)